content/javascript/concepts/strings/terms/slice/slice.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.slice()'
+Description: 'Returns a selected portion of a string.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Elements'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.slice()` method selects and returns a selected portion of a string.
+
+## Syntax
+
+```pseudo
+string.slice(start);
+
+string.slice(start, stop);
+```
+
+- The `start` index is optional and begins at zero (inclusive). Default is 0.
+- The `stop` index is also optional and defaults to the length of the string (not inclusive). Default is last element.
+
+## Example
+
+```js
+let smoothie = 'mango strawberry pineapple';
+
+let berry = smoothie.slice(6, 16);
+
+console.log(fruit);
+// Output: mango strawberry pineapple
+
+console.log(berry);
+// Output: strawberry
+```
+
```

Comment:
For example snippets, let's include a short sentence that preview a \```shell snippet with the output:

```suggestion

The output will look like the following:

```

And then the snippet would look like this:

```shell
mango strawberry pineapple
strawberry
```